core/controllers/feedback.py

```diff
@@ -158,7 +158,7 @@ class SuggestionActionHandler(base.BaseHandler):
     _ACCEPT_ACTION = 'accept'
     _REJECT_ACTION = 'reject'
 
-    @editor.require_editor
+    @acl_decorators.can_manage_suggestions_on_exploration
```

Comment:
Here this decorator can manage the arguments given here (It only needs exploration_id).

But can_edit_exploration's signature is _can_edit_exploration(self, exploration_id, escaped_state_name, **kwargs)_

Now if that's used here, thread_id would get passed as escaped_state_name which may cause an error. Is there any way to circumvent this problem (then we might use can_edit_exploration here as well)?